src/common/components/WalletBalanceInput/WalletSelectorMenu.tsx

```diff
@@ -29,15 +29,17 @@ export const WalletSelectorMenu = ({
   onChange,
   selectedWalletToken,
 }: WalletBalanceInputProps) => {
-  const selectedWalletTokenIcon = tokens.find((w) => w.symbol === selectedWalletToken.symbol)?.icon;
+  const selectedWalletTokenIcon = tokens.find(
```

Comment:
do you think we can add an indicator if there are no selected coin? i feel that it's not really clear when it's just all white

Reply:
yes that's a good idea, i added a 'Select' text if no wallet is selected by default. wdyt?
<img width="373" alt="image" src="https://user-images.githubusercontent.com/19740800/212625004-cd52e5ab-383c-4964-9c84-15ce8690247a.png">

Reply:
looks ok! thankss